bom/build.gradle.kts

```diff
@@ -2,20 +2,18 @@
  * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
  * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
+ * The ASF licenses this file to you under the Apache License, Version 2.0
```

Comment:
This is not in line with the ASF recommendation: https://www.apache.org/legal/src-headers.html#headers

Do you suggest ASF recommendations are out of date?

Reply:
I've made the headers consistent with the other headers AND consistent with the ASF recommendation (which has "to you", not "to You").

Reply:
Either I am blind or the recommendation at legal/src-headers was updated within the recent 14 hours, but I believe I did cross-check the recommendation before posting the comment.

In any case it would make sense to find out the canonical header and configure it via IDE plugin: https://github.com/vlsi/vlsi-release-plugins/blob/master/plugins/ide-plugin/src/main/kotlin/com/github/vlsi/gradle/ide/IdeExtension.kt#L64-L74

It enables to automatically configure the copyright header in IDEA, so it automatically adds the proper header as new files are created.